yt_dlp/extractor/pornhub.py

```diff
@@ -198,6 +199,20 @@ class PornHubIE(PornHubBaseIE):
             'skip_download': True,
         },
         'skip': 'This video has been disabled',
+    }, {
+        'url': 'http://www.pornhub.com/view_video.php?viewkey=ph601dc30bae19a',
+        'info_dict': {
+            'id': 'ph601dc30bae19a',
+            'uploader': 'Projekt Melody',
+            'uploader_id': 'projekt-melody',
+            'upload_date': '20210205',
+            'title': '"Welcome to My Pussy Mansion" - CB Stream (02/03/21)',
+            'thumbnail': str,
+            'duration': int,
```

Comment:
?

Reply:
I was too laxy :sweat_smile:
`duration` is int in PornHubIE anyways

Reply:
I was too lazy :sweat_smile:
`duration` is int in PornHubIE anyways